src/extend/split-contract/index.js

```diff
@@ -0,0 +1,13 @@
+export const extendCollection = async (chainId, metadata, tokenId) => {
+    metadata.id = `${metadata.contract}:${metadata.slug}`;
```

Comment:
didn't we discuss putting "courtyard-..." because just using the slug can create issue with the way we parse the id on the indexer?

Reply:
i think it should be:`${metadata.contract}:courtyard-${metadata.slug}`, it follows the same structure we are doing for opensea, soundxyz, etc.

also, instead of just adding another exend, wouldn't it make sense to support passing a "prefix" (which will be. "opensea", "courtyard", etc" and then have one extend class?

Reply:
the last point is not critical, just can be cleaner so not a blocker.